### PR TITLE
[pdata] Ran make genpdata

### PR DESCRIPTION
**Pull request by TylerHelmuth**
Regenerating pdata with `make genpdata`.  IDK if we want the comment changes, but raising the issue via this PR since the current pdata is out of sync.

### CODE REVIEW

File: pdata/internal/generated_ptrace.go [hunk 1]
```diff
@@ -1049,11 +1035,10 @@ func (es SpanLinkSlice) AppendEmpty() SpanLink {
...
 // Returns the same instance to allow nicer code like:
-//
-//	lessFunc := func(a, b SpanLink) bool {
-//	  return a.Name() < b.Name() // choose any comparison here
-//	}
-//	assert.EqualValues(t, expected.Sort(lessFunc), actual.Sort(lessFunc))
+//   lessFunc := func(a, b SpanLink) bool {
+//     return a.Name() < b.Name() // choose any comparison here
+//   }
+//   assert.EqualValues(t, expected.Sort(lessFunc), actual.Sort(lessFunc))
```
Comment (dmitryax) [inline pdata/internal/generated_ptrace.go:1041]: Hm.. it just replaces tabs with spaces in the comments. The template uses tabs. I don't think this is an improvement. Maybe is there something changed to how `go fmt` work in your setup?

Comment (TylerHelmuth) [reply pdata/internal/generated_ptrace.go:1041]: It's the output when I run `make genpdata` on main, but let me refresh my tools install and see what happens.

**Comment by bogdandrutu** [comment]
@TylerHelmuth you have a golang 1.18 the 1.19 gofmt changes this so the code is formated with 1.19

File: pdata/internal/generated_ptrace.go (hunk 1 above)
Comment (bogdandrutu) [reply pdata/internal/generated_ptrace.go:1041]: Is old go fmt that you have @TylerHelmuth

**Comment by TylerHelmuth** [comment]
gotta fix my Go version

**Pull request closed without merging**